src/lib/components/inbox/conversations/conversation_snippet.tsx

```diff
@@ -111,10 +118,10 @@ export class ConversationSnippet extends React.Component<Props, any> {
       <TouchableWithoutFeedback onPress={this.props.onSelected}>
         <Card>
           <CardContent>
-            <OpaqueImageView imageURL={imageURL} style={styles.image} />
+            <ImageView imageURL={imageURL} />
```

Comment:
It's actually `OpaqueImageView` wrapped in a styled component
so `const ImageView = styled(OpaqueImageView)`. 

I changed `OpaqueImageView` slightly to be compatible with styled components.